tests/x11regressions/libreoffice/libreoffice_mainmenu_favorites.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case 1503906 - LibreOffice: Favorite Documents link in Computer menu.
+
+use base "x11test";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    # start destop application memu
+    wait_idle;
```

Comment:
please avoid wait_idle - assert the screen you want to see. wait_idle sleeps a minute on real hardware, which is very expensive.